libnavigation-base/src/main/java/com/mapbox/navigation/base/CurrentIndicesSnapshot.kt

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.navigation.base
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+
+/**
+ * Class holding information about a snapshot of current indices.
+ * All the indices are consistent (taken from the same RouteProgress instance).
+ *
+ * @param legIndex index of a leg the user is currently on.
+ * @param routeGeometryIndex route-wise index representing the geometry point
+ * right in front of the user (see [DirectionsRoute.geometry]), null if unavailable.
+ * @param legGeometryIndex leg-wise index representing the geometry point
+ * right in front of the user (see [DirectionsRoute.geometry]), null if unavailable.
+ */
+class CurrentIndicesSnapshot internal constructor(
```

Comment:
let's move `CurrentIndicesSnapshot` under internal package as well

Reply:
They are part of the public interface, so they should remain public.

Reply:
accrording to `current.txt` it's not 🤔 may be it's not updated

Reply:
It is public API and it is in the `current.txt`... See `-base` module.

Reply:
yeah, it's self public and never set as params or method retunt value. So we can hide it

Reply:
Ah, you mean since we moved `NavigationRouterV2` to internal. If we leave it there, then yes, we can move CurrentIndicesSnapshot to internal.

Reply:
yeah, thye should be exposed in sync 👍